pumble-sdk/src/api/v1/MessagesApiClientV1.ts

```diff
@@ -54,7 +54,7 @@ export class MessagesApiClientV1 extends BaseApiClient {
     public async postEphemeral(
         channelId: string,
         payload: V1.SendMessagePayload,
-        ...toUsers: string[]
+        ...toUsers: [string, ...string[]]
```

Comment:
Wouldn't it be better separating into two params instead ? 
targetUser, otherUsers . or something like that

Reply:
Changed it

Reply:
Please check if there's some example file in the codebase to make sure all usages have been updated

Reply:
Checked it, all usages seem proper